src/settlements.py

```diff
@@ -0,0 +1,84 @@
+import sys
+import pyarrow
+
+from .geoparquet import GeoParquetWriter
+
+
+class SettlementsWriter(GeoParquetWriter):
+    """
+    The settlements layer contains point geometries representing named human
+    settlements (cities, towns, villages) and parts of settlements (boroughs,
+    qaurters, neighborhoods) from OSM.
+    """
+
+    COLUMNS = [
+        ("place", pyarrow.string()),
+        ("name", pyarrow.string()),
+        ("names", pyarrow.map_(pyarrow.string(), pyarrow.string())),
+        ("alt_name", pyarrow.string()),
+        ("alt_names", pyarrow.map_(pyarrow.string(), pyarrow.string())),
+        ("official_name", pyarrow.string()),
+        ("official_names", pyarrow.map_(pyarrow.string(), pyarrow.string())),
+        ("wikidata", pyarrow.string()),
+        ("wikipedia", pyarrow.string()),
+        ("population", pyarrow.uint32()),
```

Comment:
Nice! Yeah, I think this is a good example of using a non-stringly typed value makes the data more friendly to consumers. Realistically, nobody will ever care about a population that's not parseable as an integer.

Since this is OSM and we're all about bikeshedding edge cases... have you considered the [population of Asia](https://www.openstreetmap.org/node/36966065#map=19/51.208697/89.234375&layers=N)?

```suggestion
        ("population", pyarrow.uint64()),
```